app/assets/stylesheets/module-data.scss

```diff
@@ -433,11 +433,21 @@
     background-color: #fff;
     z-index: 10;
     top: 135%;
+    width: max-content;
+    min-width: 50rem;
```

Comment:
Has you test it in mobile/tablet? Too width to be absolute. Or missing media query